src/DataStructures/LinkedLists/utils.go

```diff
@@ -0,0 +1,78 @@
+package LinkedLists
+
+type Node struct {
+	Val  int
+	Prev *Node
+	Next *Node
+}
+
+type SinglyLinkedList struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type SinglyLinkedList should have comment or be unexported](https://app.codacy.com/manual/bumblebee211196/go-ds-daa/pullRequest?prid=5702942)